packages/mdc-chips/README.md

```diff
@@ -257,8 +257,8 @@ Mixin | Description
 `mdc-chip-trailing-icon-color($color, $opacity, $hover-opacity, $focus-opacity)` | Customizes the color of a trailing icon in a chip, optionally customizes regular/hover/focus opacities
 `mdc-chip-leading-icon-size($size)` | Customizes the size of a leading icon in a chip
 `mdc-chip-trailing-icon-size($size)` | Customizes the size of a trailing icon in a chip
-`mdc-chip-leading-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a leading icon in a chip
-`mdc-chip-trailing-icon-margin($top, $right, $bottom, $left)` | Customizes the margin of a trailing icon in a chip
+`mdc-chip-leading-icon-margin($right, $left)` | Customizes the margin of a leading icon in a chip
+`mdc-chip-trailing-icon-margin($right, $left)` | Customizes the margin of a trailing icon in a chip
```

Comment:
This is a breaking change that should be tagged in PR description.

Reply:
Done